man/fi_rxd.7.md

```diff
@@ -49,7 +48,10 @@ tested.
 
 # RUNTIME PARAMETERS
 
-No runtime parameters are currently defined.
+The *rxd* provider checks for the folowing environment variables:
+
+*FI_RXD_SPIN_COUNT*
+: Number of times to spin on progress.
```

Comment:
Please add more details on what this actually does -- e.g. Read a completion queue up to this many times until a completion is found prior to returning an EAGAIN failure to the app.  Or whatever this actually does, so that a user can figure out better how to use this and what value (1, 10, 1000000?) makes sense.